frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -244,17 +244,13 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
  * Get an icon type for the specified cluster property details (type, draft, erp, spp etc)
  */
 export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
-  const { propertyTypeId, projectWorkflow, classificationId, parcelDetail } = feature?.properties;
+  const { propertyTypeId, projectWorkflow, classificationId } = feature?.properties;
```

Comment:
parcelDetail was always usually undefined for some reason, already had the info needed anyway

Reply:
can you verify this works properly when the inventory view is open? that is normally when parcelDetail is valued.

Reply:
Looks pretty good to me I think 

https://user-images.githubusercontent.com/33818575/108283775-c4469780-7138-11eb-9bd9-871e1e6cebeb.mp4